doc/reference/files/dune-project/accept_alternative_dune_file_name.rst

```diff
@@ -0,0 +1,17 @@
+accept_alternative_dune_file_name
+---------------------------------
+
+.. dune:stanza:: accept_alternative_dune_file_name
+
+   .. versionadded:: 3.0
+
+   Specify that the alternative filename ``dune-file`` is accepted in addition
```

Comment:
```suggestion
   Specifies that the alternative filename ``dune-file`` is accepted in addition
```
All of these initial sentences are phrases rather than complete sentences. Since these are separate files, I thought they were separate pages. If they're sections in a single page and act more like a list, it might be okay with the fragments. 

I'd like to see how they look rendered. Perhaps it's reads in a way that using the initial phrase rather than a complete sentence makes sense.